docs/pages/application-access/controls.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Application Access Access Controls
+description: Role-based access control (RBAC) for Teleport Application Access
+---
+
+# Application Access Role-Based Access Control
+
+## Assigning Labels to Applications
+
+Teleport Application Access uses labels to control access to the proxied
+web applications.
+
+An application can be assigned static and dynamic labels via configuration:
```

Comment:
Nit: I found that users read active voice better: `Users can assign static and dynamic labels to apps using configuration`...